core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/JsonPath.kt

```diff
@@ -8,9 +8,9 @@ import java.io.Serializable
  * Supports just keys (in bracket notation), double quotes, arrays and wildcards.
  *
  * Examples:
- * `$["store"]["book"][*]["author"]`
+ * `[*]["author"]`
```

Comment:
same

Reply:
this one is actually not fixed

Reply:
yup! thanks, that's the danger of introducing new character-based hooks... Good find!

Reply:
Actually, this needs a fix on the doc processor side. "`\$["store"]["book"][*]["author"]`" should solve it, but it seems the "`\`" isn't removed correctly.